Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -250,16 +251,18 @@ extension SubscriptionsViewController {
                 }
 
                 groupSubscriptions?.append(unreadGroup)
+                NotificationCenter.default.post(name: Notification.Name("Unread_Message_Notification"), object: nil)
+            } else {
+                NotificationCenter.default.post(name: Notification.Name("All_Messages_Read_Notification"), object: nil)
```

Comment:
Can you use some `constant` for the notification name? Since this is being used in more than one place.